init.sh

```diff
@@ -11,26 +11,33 @@ set -e
 # - the supported test generation tools
 # - the supported code coverage tools (TODO)
 ################################################################################
+
+# Print an error message and terminate the script.
+# Takes one argument, a custom error message.
+# Prints the supplied error message and a script termination notice to stderr.
+# Terminates the script with exit code 1.
+print_error_and_exit() {
+  echo -e "${1} \nTerminating initialization... " >$2
```

Comment:
Stdout is redirected to $2, but no such argument is provided. Is this supposed to be &2?

Reply:
Yes.

Reply:
I went ahead and squashed the commits with the changes into one commit.

Reply:
> I went ahead and squashed the commits with the changes into one commit.

Thanks for the fix.
In general, please do not force-push (including squashing commits) in a pull request branch.  Doing so leaves the code review comments attached to a commit that no longer exists.  This can make the pull request harder to work with.